utils/cliutils/commandsflags.go

```diff
@@ -1151,6 +1153,14 @@ var flagsMap = map[string]cli.Flag{
 		Name:  target,
 		Usage: "[Default: ./] Path for downloaded update files.` `",
 	},
+	DBSyncV3: cli.BoolFlag{
+		Name:  DBSyncV3,
+		Usage: "[Optional] Use Xray DBSync V3",
+	},
+	PeriodicDBSyncV3: cli.BoolFlag{
+		Name:  PeriodicDBSyncV3,
+		Usage: fmt.Sprintf("[Optional] Get Xray DBSync V3 Periodic Package (Use with %s flag)", DBSyncV3),
```

Comment:
...(Use with %s flag)... --> ...(Use with the --%s command option)...

Also, why is it necessary to have these two separate options? Is `DBSyncV3` sometimes used without `PeriodicDBSyncV3`?

Reply:
The DBSync V3 Feature has 2 states:

```--dbsyncv3 ```

Will download the full on-boarding package containing all the data (as same as happening now, when you omit the ```--from``` and ```--to``` flags)

and

```--dbsyncv3 --periodic ```

Will download a periodic update package. This is replacing the ```--from``` and ```--to``` flags.

So to summarize:
when full on-boarding  we will use ```--dbsyncv3``` 
when doing regular updates we will use ```--dbsyncv3 --periodic```